### PR TITLE
[FIX] PEP8 and Flake

### PR DESCRIPTION
**Pull request by oscarolar**
Modified the code to pass all tests on travis and some other CI platforms.

### CODE REVIEW

**Comment by oscarolar** [comment]
Please do NOT MERGE, found some extra issues, fixing now.

**Comment by oscarolar** [comment]
nevermind, please Merge when you consider.

**Comment by oscarolar** [comment]
Sorry for bothering you, 
is there any reason of why you havent merged this PR?, if there is, I will fix it with no problem, thanks.

**Comment by JayVora-SerpentCS** [comment]
No reason brother! Its just I haven't been able to spend time on this.

File: website_product_brand/ChangeLog.txt [hunk 1]
```diff
@@ -4,3 +4,6 @@ Maintaining the changelog of the module for each commit.
 
 1.0 : Vimesh Chaudhari - Serpent Consulting Services Pvt Ltd.
 	* Added the module.
+1.1 : Oscar Alcala - Vauxoo
+	* fixed Pep8 and Pylint on code.
+
```
Comment (JayVora-SerpentCS) [inline website_product_brand/ChangeLog.txt]: Set your changelog in reverse order, the recent version should come on top.

**Comment by JayVora-SerpentCS** [comment]
Ready to merge once you update changelog.

**Pull request merged**